docs/source/overview/quicktour.md

```diff
@@ -115,7 +114,9 @@ The engine which handles the auto-tuning process. You can select search strategy
 
 ```json
 "engine": {
-    "cache_dir": ".cache"
+    "host": {"type": "LocalSystem"},
```

Comment:
this line looks different with the example https://github.com/microsoft/Olive/blob/main/examples/resnet_ptq_cpu/resnet_config.json#L101

Do we support both of them?

Reply:
yes, `host` can refer to a name in the `systems` list or be a system config dict.